src/pages/About/About.jsx

```diff
@@ -0,0 +1,175 @@
+import React from 'react';
+import { Box, Container, Link, Paper, Typography } from '@mui/material';
+import customColors, { defaultTheme } from '../../assets/styles';
+import c from '../../App.module.css';
+import Team from './Team';
+
+const paragraphStyle = {
+    textAlign: 'justify',
+    textIndent: '2em',
+    mb: 2,
+};
+
+const About = ({ team }) => {
+    return (
+        <>
+            {/* Title section*/}
+            <Container
+                sx={{
+                    backgroundColor: customColors.backgroundLight,
+                    maxWidth: 'none !important',
+                    padding: '50px 0 20px 0',
+                }}
+            >
+                <Typography
+                    variant={'h5'}
+                    sx={{
+                        textTransform: 'uppercase',
+                        textAlign: 'center',
+                        fontWeight: 'bold',
+                        [defaultTheme.breakpoints.down('md')]: {
+                            fontSize: '20px',
+                        },
+                    }}
+                >
+                    About us
+                </Typography>
+            </Container>
+            {/* Content section */}
+            <Box
+                sx={{
+                    backgroundColor: customColors.backgroundLight,
+                    maxWidth: 'none !important',
+                    padding: '0 0 120px 0',
+                }}
+            >
+                <Box
+                    className={c.container}
+                    sx={{ justifyContent: 'start', textAlign: 'justify' }}
+                >
+                    <Typography variant={'body1'} sx={paragraphStyle}>
+                        Nowadays, getting hired as a junior developer is quite a tedious
+                        task.
+                    </Typography>
+                    <Typography variant={'body1'} sx={paragraphStyle}>
```

Comment:
I'm seeing a big space between sentences here 
![image](https://github.com/Code-the-Dream-School/ee-prac-team3-front/assets/6400323/0b213e68-c366-4195-a824-8b60932f4558)

Reply:
Done ✅